### PR TITLE
Hide various page elements for A/B/C test

### PR DESCRIPTION
**Pull request by timEulitz**
Revert this commit later via `git revert 43009994d363f1dc26770b6c19934bcd95879bbd`.
https://phabricator.wikimedia.org/T226387

### CODE REVIEW

File: skins/cat17/templates/partials/Page_Footer.html.twig [hunk 1]
```diff
@@ -17,7 +17,7 @@
 			</div>
 			<nav class="menu menu-footer col-xs-12 col-sm-4 col-md-4">
 				<ul class="list-menu list-unstyled">
-					<li>
```
Comment (tzhelyazkova) [inline skins/cat17/templates/partials/Page_Footer.html.twig]: Wouldn't that be ready in `laika` by the time we run the three skins test?

File: skins/laika/src/components/layout/Header.vue [hunk 2]
```diff
@@ -43,7 +43,7 @@ export default Vue.extend( {
 			'headerMenu': [
 				{ id: 'donation-form', localeId: 'donate', url: '/' },
 				{ id: 'membership-application', localeId: 'membership_application', url: '/apply-for-membership' },
-				{ id: 'faq-page', localeId: 'faq', url: '/faq' },
+				// T226387 { id: 'faq-page', localeId: 'faq', url: '/faq' },
 				{ id: 'use-of-funds', localeId: 'use_of_resources', url: '/use-of-funds' },
```
[the remark this answers is not in the document]
Comment (timEulitz) [reply skins/laika/src/components/layout/Header.vue]: I assume this is done under the assumption that we will finish it in the next 20 days :male_detective:.

[the remark this answers is not in the document]
Comment (tzhelyazkova) [reply skins/laika/src/components/layout/Header.vue]: But what's the point of disabling use of funds in cat17 then?

File: skins/cat17/templates/partials/Page_Footer.html.twig (hunk 1 above)
Comment (timEulitz) [reply skins/cat17/templates/partials/Page_Footer.html.twig]: I thought the same thing, the requirements dont fully make sense to me to be honest.

File: skins/laika/src/components/layout/Header.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (timEulitz) [reply skins/laika/src/components/layout/Header.vue]: I also don't get it, same for the FAQ in Laika but not in cat17. Gonna check with Till.

**Comment by timEulitz** [comment]
@tzhelyazkova Talked to Till, we're dumping the ticket for now. The point was to disable pages which will not be ready for the ABC test but since we still have three weeks to go, we will just do that in the week of the ABC test with the complete picture of what will go live or not.

**Pull request closed without merging**